src/login.rs

```diff
@@ -17,6 +17,10 @@ pub struct LoginCLIConfig {
     /// Password
     #[clap(long, env = "TUTANOTA_CLI_PASSWORD")]
     password: String,
+    
+    /// One-Time-Code
+    #[clap(long, default_value_t = 0.to_string(), env = "TUTANOTA_CLI_ONETIMECODE")]
+    onetimecode: String,
```

Comment:
```suggestion
    #[clap(long, env = "TUTANOTA_CLI_ONETIMECODE")]
    onetimecode: Option<String>,
```

Should be optional. clap will default this to `None` and not complain if the env var (or the CLI parameter) is missing.